packages/flutter/lib/src/painting/image_stream.dart

```diff
@@ -42,19 +44,23 @@ class ImageInfo {
   /// (e.g. in the arguments given to [Canvas.drawImage]).
   final double scale;
 
+  /// A string used for debugging purpopses to identify the source of this image.
+  final String tag;
```

Comment:
"tag" sounds to me like "label". The value that we've assigned to it is more like "source". If we document it as "identify the source", why not just call it "source" or "debugSource"? Forgive me if my English understanding is limited...

Reply:
I can rename this. I think I was originally expecing this to potentially be an object that might just hold the image cache key.